README.md

```diff
@@ -107,6 +107,11 @@ require'nvim-treesitter.configs'.setup {
   highlight = {
     enable = true,              -- false will disable the whole extension
     disable = { "c", "rust" },  -- list of language that will be disabled
+    -- Setting this to true will run `:h syntax` and tree-sitter at the same time.
+    -- Set this to `true` if you depend on 'syntax' being enabled (like for indentation).
+    -- Using this option may slow down your editor, and you may see some duplicate highlights.
+    -- Instead of true it can also be a list of languages
+    additional_vim_regex_highlighting = false,
```

Comment:
This is already in the highlight module example, not sure if it is worth duplicating

Reply:
I would always put all options of module. We could shorten the docs but I thought that a lot of users only would use the highlighting feature without reading further-